commands/host.vegman

```diff
@@ -193,35 +234,75 @@ function cmd_virtualmedia_mount {
     return 1
   fi
 
-  if [[ ! -e ${imagefile} ]] || [[ -d ${imagefile} ]]; then
-    echo "Please specify the path to image file" >&2
+  if ! [[ ${imagefile} =~ ${imagefile_pattern} ]]; then
+    echo "Please specify the valid path to the image file" >&2
     return 1
   fi
 
-  usb-ctrl insert "usb.ms.cli" "${imagefile}" "${type}"
+  if [[ ! -z ${username} ]] || [[ ! -z ${password} ]]
+  then
+    local cred_pipe_fifo=$(mktemp -u)
+    cred_pipe=6
+    mkfifo $cred_pipe_fifo
+    exec 6<>$cred_pipe_fifo
```

Comment:
You don't need to hardcode this value. The line 245 is excess.
Use:
```bash
exec {cred_pipe}<>${cred_pipe_fifo}
```
and `${cred_pipe}` will contain the actual value.

Reply:
yes, I know, but 
```
:~$ pipe_id=6
:~$ cred_pipe_fifo=$(mktemp -u)
:~$ mkfifo $cred_pipe_fifo
:~$ exec $pipe_id<>$cred_pipe_fifo
bash: exec: 6: not found
```
and `exec 6<>$cred_pipe_fifo` is success

Reply:
Ahh, I saw. Should be `{cred_pipe}`, not `${cred_pipe}`

Reply:
I checked script with `exec {cred_pipe}<>${cred_pipe_fifo}` and the suggested way doesn't works.

Reply:
I just checked. I created a file `x`:

```
#!/bin/bash
cred_pipe_fifo=$(mktemp -u)
mkfifo $cred_pipe_fifo
exec {cred_pipe}<>${cred_pipe_fifo}
echo Fifo is at ${cred_pipe}
```

and ran it:

```
$ chmod 755 ./x
$ ./x
Fifo is at 10
$
```

Reply:
I have a little bit confused... For a bash terminal, it's doesn't worked. But in the script is it.
Ok, I changed `6` to `{cred_pipe}`.